package.json

```diff
@@ -24,12 +25,14 @@
     "url": "https://github.com/cimasim89/flat-file-creator/issues"
   },
   "dependencies": {
+    "@types/lodash": "^4.14.151",
```

Comment:
Hm, I committed this suggestion without thinking too much about it, but now I'm starting to feel more strongly about opening it up to compatible versions. This is because our versioning will affect the versions decided in upstream projects. I know npm permits multiple versions to co-exist, but that has been very problematic as more projects start to accept typescript because typescript will use the first definitions that it finds, and that can create a situation in which your project won't compile even though the underlying code works.

While that's technically a problem specifically with typescript, I consider it safer to encourage the use of a single version of a dependency, rather than multiple versions, so I'd like to suggest that we open this back up with the "compatible" version operator once again. Would you be willing to consider that?